Gemfile.lock

```diff
@@ -1,30 +1,10 @@
 GIT
   remote: https://github.com/18F/uswds-jekyll.git
-  revision: 413413405c01ffbbb039ab7bd38767db9ab87484
+  revision: 472991d0f798968dfc0fb4f7c096a0061f89f8cd
   specs:
     uswds-jekyll (4.2.0)
       jekyll (>= 3.4, < 5)
 
-GIT
-  remote: https://github.com/jekyll/jekyll.git
-  revision: 3e440a9cc10cf51a6b140ecdf4b4a70f1aed649e
-  specs:
-    jekyll (4.0.0)
-      addressable (~> 2.4)
-      colorator (~> 1.0)
-      em-websocket (~> 0.5)
-      i18n (>= 0.9.5, < 2)
-      jekyll-sass-converter (~> 2.0)
-      jekyll-watch (~> 2.0)
-      kramdown (~> 2.1)
-      kramdown-parser-gfm (~> 1.0)
-      liquid (~> 4.0)
-      mercenary (~> 0.4.0)
-      pathutil (~> 0.9)
-      rouge (~> 3.0)
-      safe_yaml (~> 1.0)
-      terminal-table (~> 1.8)
-
```

Comment:
Why was this removed?

Reply:
It wasn't removed, so much as it _moved_—see `jekyll` below.

The long version: the `Gemfile.lock` tracks dependencies from Git (`GIT`) separately from those from rubygems.org (`GEM`), so changing the `Gemfile` like so:

```diff
-gem 'jekyll', github: 'jekyll/jekyll'
+gem 'jekyll'
```

caused this. Does that make sense?